### DISCUSSION TITLE
Nightingale cannot import a dashboard

### DISCUSSION THREAD

**Discussion by ljw999** [ccfos/nightingale#1319] [Q&A]
导入就会提示报错，Error 1054: Unknown column 'ident' in 'field list'，谁知道这个怎么解决吗

**Comment by jsers** [comment]
描述详细点，是哪个类型的导入，内置大盘的话说下哪个大盘， Grafana 大盘的话贴下 JSON
[image: image.png]

罗经伟 ***@***.***> 于2022年12月12日周一 18:04写道：

**Comment by ljw999** [comment]
![image](https://user-images.githubusercontent.com/87891114/207208339-e9ce6d51-59b9-4ccd-8a59-59e63125e00a.png)
我导入的是系统的监控大盘，但是不止导入这个大盘有问题，导入自带的大盘都会有问题
![image](https://user-images.githubusercontent.com/87891114/207208509-173ba864-30f1-4535-9a73-e41bf93b9954.png)
是对客户端还是服务端版本啥的有要求呢
前端版本：5.13.2
后端版本：v5.13.1
还是前后端版本不一致导致的

**Comment by ljw999** [comment]
我找到问题了，需要修改sql，在release页面有每个版本对应的 sql
